packages/office-ui-fabric-react/src/components/Slider/Slider.test.tsx

```diff
@@ -28,14 +28,15 @@ describe('Slider', () => {
     const sliderLine = wrapper.find('.ms-Slider-line');
     const sliderThumb = wrapper.find('.ms-Slider-slideBox');
 
-    sliderLine.getDOMNode().getBoundingClientRect = () => ({
-      left: 0,
-      top: 0,
-      right: 100,
-      bottom: 40,
-      width: 100,
-      height: 40
-    });
+    sliderLine.getDOMNode().getBoundingClientRect = () =>
+      ({
+        left: 0,
+        top: 0,
+        right: 100,
+        bottom: 40,
+        width: 100,
+        height: 40
+      } as DOMRect);
```

Comment:
Quick clarification q - why specifically is this a cast to DOMRect? Didn't find anything about DOMRect on https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#dom-changes

Reply:
Because it generated a type error as it was. If you click through the link in your link, it appears `getBoundingClientRect` return type changed as part of this:

![image](https://user-images.githubusercontent.com/26070760/72469633-a91a9300-3794-11ea-8197-88f9dc43ebde.png)

These are tests and they work as is, so I think it's acceptable to use a cast in these cases.